pkg/apis/toolchain/v1alpha1/masteruserrecord_types.go

```diff
@@ -25,6 +27,8 @@ const (
 	MasterUserRecordUnableToDeleteUserAccountsReason         = "UnableToDeleteUserAccounts"
 	MasterUserRecordUnableToRemoveFinalizerReason            = "UnableToRemoveFinalizer"
 	MasterUserRecordDisabledReason                           = disabledReason
+	MasterUserRecordUserProvisionedNotificationCreatedReason = "Created"
+	MasterUserRecordUserProvisionedNotificationFailedReason  = "Failed"
```

Comment:
I believe that you could make it shorter - just 
```suggestion
	MasterUserRecordNotificationCRCreatedReason = "NotificationCRCreated"
	MasterUserRecordNotificationCRCreationFailedReason  = "NotificationCRCreationFailed"
```
or make it generic: 
```suggestion
	NotificationCRCreatedReason = "NotificationCRCreated"
	NotificationCRCreationFailedReason  = "NotificationCRCreationFailed"
```
and move it out of the masteruserrecord_types.go file

Reply:
this looks much better! i guess for now, let's leave them here since they aren't used anywhere else, when we add the other notifications we can always move it and make it more generic. i've made the change here: c9659c3889827f95489a520d361e4c78df50d9a5